allauthdemo/settings.py

```diff
@@ -0,0 +1,341 @@
+# timezone as the operating system.
+# If running in a Windows environment this must be set to the same as your
+# system time zone.
+#
+# TIME_ZONE = 'Europe/London'
+# TIME_ZONE = 'America/Adak'
+# TIME_ZONE = 'Africa/Abidjan'
+# TIME_ZONE = 'Antarctica/Casey'
+# TIME_ZONE = 'Arctic/Longyearbyen'
+# TIME_ZONE = 'Atlantic/Reykjavik'
+# TIME_ZONE = 'GMT'
+
+TIME_ZONE = 'UTC'
+
+# Language code for this installation. All choices can be found here:
+# http://www.i18nguy.com/unicode/language-identifiers.html
+LANGUAGE_CODE = 'en-GB'
+
+# check site id in admin
+SITE_ID = 1
+
+# If you set this to False, Django will make some optimizations so as not
+# to load the internationalization machinery.
+USE_I18N = True
+
+# If you set this to False, Django will not format dates, numbers and
+# calendars according to the current locale
+USE_L10N = True
+
+USE_TZ = True
+
+# Absolute filesystem path to the directory that will hold user-uploaded files.
+# Example: '/home/media/media.lawrence.com/media/'
+# If using manage.py runserver" the resulting server will not deliver media files 
+# unless DEBUG = True.
+# media is served locally in this example
+MEDIA_ROOT = os.path.join(BASE_DIR, 'media')
+
+# URL that handles the media served from MEDIA_ROOT. Make sure to use a
+# trailing slash.
+# Examples: 'http://media.lawrence.com/media/', 'http://example.com/media/'
+# media is served locally in this example
+MEDIA_URL = '/media/'
+
+# Absolute path to the directory static files should be collected to.
+# Don't put anything in this directory yourself; store your static files
+# in apps' 'static/' subdirectories and in STATICFILES_DIRS.
+# Example: '/home/media/media.lawrence.com/static/'
+STATIC_ROOT = os.path.join(BASE_DIR, 'static')
+
+# URL prefix for static files.
+# Example: 'http://media.lawrence.com/static/'
+STATIC_URL = '/static/'
+
+# URL prefix for admin static files -- CSS, JavaScript and images.
+# Make sure to use a trailing slash.
+# Examples: 'http://foo.com/static/admin/', '/static/admin/'.
+
+# Additional locations of static files
+STATICFILES_DIRS = (
+
+    # Put strings here, like '/home/html/static' or 'C:/www/django/static'.
+    # Always use forward slashes, even on Windows.
+    # Don't forget to use absolute paths, not relative paths.
+    
+    os.path.join(BASE_DIR, 'allauthdemo/static'),
+)
+
+# List of finder classes that know how to find static files in
+# various locations.
+STATICFILES_FINDERS = (
+    'django.contrib.staticfiles.finders.FileSystemFinder',
+    'django.contrib.staticfiles.finders.AppDirectoriesFinder',
+)
+
+#STATICFILES_STORAGE = 'whitenoise.storage.CompressedManifestStaticFilesStorage'
+#DEFAULT_FILE_STORAGE = 'storages.backends.s3boto3.S3Boto3Storage'
+
+STATICFILES_STORAGE = 'django.contrib.staticfiles.storage.StaticFilesStorage'
+DEFAULT_FILE_STORAGE = 'django.core.files.storage.FileSystemStorage'
+
+
+MIDDLEWARE = (
+    #'whitenoise.middleware.WhiteNoiseMiddleware',
```

Comment:
Uncomment this line so that whitenoise works with heroku

Reply:
As above